packages/mdl-radio/mdl-radio.scss

```diff
@@ -0,0 +1,108 @@
+$mdl-radio-size: 20px !default;
+$mdl-radio-foreground-color: #333 !default;
+$mdl-radio-accent-color: rgba(255,82,82,0.87) !default;
+$mdl-radio-disabled-color: gray !default;
+$mdl-radio-timing-fn: ease 280ms !default;
+
+// Top-level host container.
+.mdl-radio {
+  color: $mdl-radio-foreground-color;
+  cursor: pointer;
+  display: inline-block;
+  transition: color $mdl-radio-timing-fn;
+}
+
+// Container for radio circles and ripple.
+.mdl-radio__container {
```

Comment:
Usually (and by "usually" I mean the few files I wrote 😛) the standard approach has been to nest element classes within the top-level host container, and keep modifier classes at the top-level. I'm totally cool deviating from this approach, but do you think it should be standardized?